spec/acceptance/dns_spec.rb

```diff
@@ -29,4 +29,17 @@
   describe command('dig +short SOA example.com @localhost') do
     its(:stdout) { is_expected.to match("ns1.example.com. root.example.com. 1 86400 3600 604800 3600\n") }
   end
+
+  before(:context) do
+    describe command('dnf remove -y bind bind-utils') do
+      its(:stdout) { should match /Removed:/ }
+    end
```

Comment:
This assumes dnf is present, but it's not everywhere. I think you can use:
```suggestion
    on hosts, puppet('resource', 'package', 'bind', 'ensure=purged')
    on hosts, puppet('resource', 'package', 'bind-utils', 'ensure=purged')
```